src/Mailer/Email.php

```diff
@@ -2084,10 +2085,20 @@ protected function _logDelivery($contents)
         }
         Log::write(
             $config['level'],
-            PHP_EOL . $contents['headers'] . PHP_EOL . PHP_EOL . $contents['message'],
+            PHP_EOL . $this->_isKeyArray($contents['headers']) . PHP_EOL . PHP_EOL . $this->_isKeyArray($contents['message']),
             $config['scope']
         );
     }
+    
+    
+    /**
+     * @param null $key
+     * @return null|string
+     */
+    private function _isKeyArray($key = null)
```

Comment:
`_is*` prefix suggests that the method returns `bool`. 
What about `_flatten` for a  method name?

Reply:
And never use private, use protected, for these cases.

Reply:
Why not private? I'm not sure if that method will ever be used again :)
And yes it actually suggests a boolean didn't realised that at the time.

Reply:
> I'm not sure if that method will ever be used again :)

Exactly, thus from a a framework perspective allowing.